docs/references/errors.md

```diff
@@ -51,57 +51,58 @@ message GreetingRequest {
 }
 ```
 
-## META0003
+## META0003 {#META0003}
 
-Cannot reach the endpoint to execute service discovery. Make sure:
+Cannot reach the deployment to execute service discovery. Make sure:
```

Comment:
I think we should either use this term in the entire docs or stick to endpoint until we do the complete switch.
Also, if I read `deployment` the first thing I think is my Restate cluster deployment, but maybe that is just me.

Reply:
Especially, as long as we have this page called `deployment`: https://docs.restate.dev/restate/deployment

Reply:
This is something that was auto generated from the error descriptions. Maybe I can just remove the respective commit from the PR.

Reply:
I'll change it to "service deployment".